kas/demos/raspberrypi4-64-wifi.yml

```diff
@@ -0,0 +1,19 @@
+header:
+  version: 14
+  includes:
+  - kas/raspberrypi4-64.yml
+
+repos:
+  meta-mender-community:
+    layers:
+      meta-mender-update-modules:
+      kas/demos/meta-mender-raspberrypi-wifi:
+
+local_conf_header:
+  wifi: |
+    DEMO_WIFI_SSID = "Demo_SSID"
+    DEMO_WIFI_PASSKEY = "Demo_Password"
+    LICENSE_FLAGS_ACCEPTED = "synaptics-killswitch"
```

Comment:
Is it allowed to preemptively add this for someone else..? I thought this had to be deliberate act, legally speaking. I'm not totally sure though.

Reply:
If it's not set in this way, the `kas` build is non-functional, so also for the NXP based stuff which requires also an EULA being accepted I went the same route. But it should be pointed out verbatim, that its true.